package ThreadClasss.RunnableThread.ThrouwsExeceptionEx;

public class MainRunnable2 {
    
}
